NeuralABC_tools/arrays.py

```diff
@@ -37,33 +37,47 @@ def map_vals_to_index(index_array, key_vals):
     return key_vals[index]  # fill key_vals into index
 
 
-def summed_statistics_by_index(index_array, val_array, statistic='mean', ignore_zero_index=True):
+def bin_statistics_by_index(index_array, val_array, statistic='mean', ignore_zero_index=True, remove_nans=True):
     """
-    Compute summary statistics on val_array based grouped by indices in index_array.
-    This is a basic wrapper for scipy.stats.binned_statistic.
+    Wrapper for scipy.stats.binned_statistic
```

Comment:
would be good to add more of a function definition here to clarify what this is doing.

Reply:
I updated the docstring to make clearer, and clarified what the function returns. 
291255b4991f0c1b10cc2261c435e8ecdffe7bc9